Dockerfile

```diff
@@ -1,33 +1,22 @@
-FROM opensuse/leap:15.2
+FROM opensuse/tumbleweed
```

Comment:
IMHO, I would stick to a certain release

Reply:
I think when we created branch crmsh-4.4, then we change the image based on leap:15.4, that seems make more sense